README.md

```diff
@@ -1,24 +1,182 @@
 
-* ...
+- Then go to this [link](http://localhost:3000/api-docs).
+
+- You should see the below page in your browser:
+
+![App Screenshot](app/assets/api-doc-1.png)
+
+> Before requesting any other option, you must create a user with this sign-up endpoint. After clicking to 'Try it Out' button, you can type your username & password right between quote signs and press 'Execute'.
+
+![App Screenshot](app/assets/api-doc-create-user.png)
+
+> After creating a user; you will receive a bearer authentication token as a response of this request if your user credentials are correct & valid.
+
+> The response should be like this; if your user is successfully created.
+
+![App Screenshot](app/assets/api-doc-token.png)
+
+> To learn more about bearer authentication; please check this [ link ](https://swagger.io/docs/specification/authentication/bearer-authentication/).
+
+> Once you got your token, you need to authenticate your by adding it to your requests' header. Copy this token and click to below authenticate button.
+
+![App Screenshot](app/assets/api-doc-3.png)
+
+> Paste your token to below text box. Press 'Authorize' and close the pop-up menu.
+
+![App Screenshot](app/assets/api-doc-4.png)
+
+> These are the all endpoints that you can consume after adding your token in your header for each GET/POST/DELETE request.
+
+![App Screenshot](app/assets/api-doc-2.png)
+
+## React Frontend
+
+> You can check the dedicated front end for this API with below links.
+
+[React Frontend repository](https://github.com/yigitm/book-appointment-frontend/tree/Readme-Frontend)
+
+![React Frontend](app/assets/app.gif)
+
+## Built With
+
+- Ruby
+- Rails
+- JWT
+- Bcrpyt
+- Rspec
+- Rswag
+- YAML
+- Rubocop
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Ruby should be installed in your local machine. Please check this [article](https://www.theodinproject.com/lessons/ruby-installing-ruby) how to install/setup Ruby for different operating systems.
+
+Ruby on Rails should be installed in your local machine. Please check this [article](https://www.theodinproject.com/lessons/ruby-on-rails-installing-rails) how to install/setup Ruby for different operating systems.
+
+### Setup
+
+To get a local copy of the repository please run the following commands on your terminal.
+
+If you are not familiar to use terminal, please check this [article](https://www.theodinproject.com/courses/web-development-101/lessons/command-line-basics-web-development-101)
+
+### Install
+
+Open your terminal & go to your desktop.
+
+```
+cd your_desktop
+```
+
+Run below commands.
+
+```
+git clone https://github.com/yigitm/book-appointment.git
+```
+
+### Usage
+
+Go to the app folder.
+
+```
+cd book-appointment
+```
+
+Run below command to install packages that app is depended to run.
+
+```
+bundle install
+```
+
+Run below to set database.
+
+```
+rails db:create
+rails db:migrate
+rails db:seed
```

Comment:
When I run this, I get following error:
![image](https://user-images.githubusercontent.com/31193823/167973613-fcd75bbf-a4f8-4adb-919e-46e96b54d1b9.png)

Reply:
check how the seeds are created - see if maybe you are missing something in instructions before seeding

Reply:
Thank you. I totally missed that, I was using these seeds at beginning of the project. I removed them.